src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseChecklistServiceTests.cs

```diff
@@ -87,10 +92,20 @@ await using (var context = ContentDbUtils.InMemoryContentDbContext(contextId))
                 await context.SaveChangesAsync();
             }
 
+            var dataImportService = new Mock<IDataImportService>(MockBehavior.Strict);
+            var metaGuidanceService = new Mock<IMetaGuidanceService>(MockBehavior.Strict);
+            var methodologyRepository = new Mock<IMethodologyRepository>(MockBehavior.Strict);
+            var releaseDataFileRepository = new Mock<IReleaseDataFileRepository>(MockBehavior.Strict);
+
             await using (var context = ContentDbUtils.InMemoryContentDbContext(contextId))
             {
-                var releaseDataFileRepository = new Mock<IReleaseDataFileRepository>();
-
+                methodologyRepository
+                    .Setup(mock => mock.GetLatestMethodologiesByRelease(release.Id))
+                    .ReturnsAsync(new List<Methodology>
```

Comment:
Ooh just think, when you can write `.ReturnsAsync(AsList(methodology));` instead :)